light-client/examples/light_client.rs

```diff
@@ -160,19 +160,20 @@ fn sync_cmd(opts: SyncOpts) {
         ProdEvidenceReporter::new(peer_addr),
     );
 
-    let mut handle = supervisor.handle();
+    let handle = supervisor.handle();
 
     std::thread::spawn(|| supervisor.run());
 
     loop {
-        handle.verify_to_highest_async(|result| match result {
+        let maybe_block = handle.verify_to_highest();
+        match maybe_block {
             Ok(light_block) => {
-                println!("[ info  ] synced to block {}", light_block.height());
+                println!("[info] synced to block {}", light_block.height());
             }
-            Err(e) => {
-                println!("[ error ] sync failed: {}", e);
+            Err(err) => {
+                println!("[error] sync failed: {}", err);
```

Comment:
Out of curiosity: what triggered removing the whitespaces here?

Reply:
Looked like a mistake, what would be the reason for it?

Reply:
As long as this consistent throughout the codebase, I don't really care. @romac would know if there was any particular reason. I don't know and can't think of any reason.